contracts/ocr2/src/integration_tests.rs

```diff
@@ -439,23 +441,27 @@ fn transmit_happy_path() {
 
     // use a new set of keypairs and signers
     let mut keypairs = Vec::new();
-    for _ in 0..4 {
+    for _ in 0..19 {
         let sk = SigningKey::new(thread_rng());
         keypairs.push(sk);
     }
     let signers = keypairs
         .iter()
-        .map(|sk| VerificationKeyBytes::from(sk).as_ref().to_vec())
+        .map(|sk| Binary(VerificationKeyBytes::from(sk).as_ref().to_vec()))
         .collect();
 
     let msg = ExecuteMsg::SetConfig {
         signers,
         transmitters: env.transmitters.clone(),
-        f: 1,
-        onchain_config: vec![],
+        f: 6,
+        onchain_config: Binary(vec![]),
         offchain_config_version: 2,
-        offchain_config: vec![5, 5, 6],
+        offchain_config: Binary(vec![1; 2165]),
     };
+
+    const MAX_MSG_SIZE: usize = 4 * 1024; // 4kb
```

Comment:
We should find out how much overhead does Serum mutlisig exactly adds, and consider it here while testing the tx size limit with the max oracles number. 

@ebarakos can you please research this and add it in?

Reply:
If we go with the "approvable proposal by digest" multisig proposal, then this suggestion won't be relevant anymore.